src/components/admin/create-meeting.js

```diff
@@ -73,7 +74,8 @@ function CreateMeeting() {
 
     function handleTextInputChange(event) {
         const value = event.target.value
-        setMeetingId(value)
+        // setMeetingId(value)
```

Comment:
Hey, can we please remove these commented out lines? For the rest of the files as well. Thanks!